crates/nu-command/src/filesystem/utouch.rs

```diff
@@ -149,9 +150,59 @@ impl Command for UTouch {
             if file_glob.item.as_ref() == "-" {
                 input_files.push(InputFile::Stdout);
             } else {
-                let path =
+                let file_path =
                     expand_path_with(file_glob.item.as_ref(), &cwd, file_glob.item.is_expand());
-                input_files.push(InputFile::Path(path));
+
+                if !file_glob.item.is_expand() {
+                    input_files.push(InputFile::Path(file_path));
+                    continue;
+                }
+
+                let mut expanded_globs = glob(&file_path.to_string_lossy())
+                    .unwrap_or_else(|_| {
+                        panic!(
+                            "Failed to process file path: {}",
+                            &file_path.to_string_lossy()
+                        )
+                    })
+                    .peekable();
+
+                if expanded_globs.peek().is_none() {
+                    let file_name = file_path.file_name().unwrap_or_else(|| {
+                        panic!(
+                            "Failed to process file path: {}",
+                            &file_path.to_string_lossy()
+                        )
+                    });
+
+                    if file_name.to_string_lossy().contains("*")
+                        || file_name.to_string_lossy().contains("?")
+                        || file_name.to_string_lossy().contains("{")
```

Comment:
I don't think we currently expand globs with `{}`

![image](https://github.com/user-attachments/assets/e6db6c38-e3b0-4b07-8594-95e7a012b3a9)